_data/simple-icons.json

```
@@ -15536,6 +15536,11 @@
             "hex": "EA7100",
             "source": "https://www.xda-developers.com"
         },
+        {
+            "title": "Xendit",
+            "hex": "446CB3",
```

Also, from the badges on that branding page, it looks like the colour might actually be `#4573FF` - the different blue in the SVG you used could be a result of converting CMYK to RGB. What do you think?
```suggestion
            "hex": "4573FF",
```

---

I've taken that HEX from their colored [svg logo](https://github.com/simple-icons/simple-icons/assets/54018894/4b5d79be-65e6-4632-8937-e57fb8b4c335) found in their logo kit.

---

I got `#4573FF` when checking their badges and website iconography. `#4373FC` is also the color used for the header background, a lot of button background and links font color.

Personally, I would opt for this shade of blue as their brand color instead of `#446CB3`.